system/src/Grav/Common/Page/Page.php

```diff
@@ -215,17 +215,17 @@ public function translatedLanguages($onlyPublished = false)
      */
     public function untranslatedLanguages($includeUnpublished = false)
     {
-        $filename = substr($this->name, 0, -(strlen($this->extension())));
-        $config = Grav::instance()['config'];
-        $languages = $config->get('system.languages.supported', []);
+        $filename              = substr($this->name, 0, -(strlen($this->extension())));
```

Comment:
why all the extra space? it was fine before. if you do not know exactly what psr2 is then read up on it [http://www.php-fig.org/psr/psr-2/](here) to ease your work you can install some linters for php aswell as. https://github.com/squizlabs/PHP_CodeSniffer and https://phpmd.org/ are useful. If you are using phpstorm you can easily set up code style and lint automatically from Code->Reformat code. Then you do not have to guess what is wanted from you and let the tools point it out.